src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -200,10 +204,15 @@ export class EmbeddablePanel extends React.Component<Props, State> {
   };
 
   private getActionContextMenuPanel = async () => {
-    const actions = await this.props.getActions(CONTEXT_MENU_TRIGGER, {
+    let actions = await this.props.getActions(CONTEXT_MENU_TRIGGER, {
       embeddable: this.props.embeddable,
     });
 
+    const { disabledActions } = this.props.embeddable.getInput();
+    if (disabledActions) {
```

Comment:
chore: should we also check for .length === 0 here.

Reply:
I don't think so, if `disabledActions` is provided, it will always have something inside, otherwise that property would not be provided. But even if an empty array is provided, I'm not sure I would want to add an extra condition to check if it is empty.